src/pages/CoinPage/index.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default class CoinPage extends React.Component {
+  state = {};
```

Comment:
It doesn't look like this component needs state and could be converted to a functional component (unless you have plans for it in the future - then just ignore this comment (: ).

Reply:
It will definitely need state lol